js/exporter/jspdf/v3/rows_generator.js

```diff
@@ -43,7 +43,8 @@ function generateRowsInfo(dataProvider, dataGrid, headerBackgroundColor) {
                 columns,
                 rowType,
                 colCount: columns.length,
-                backgroundColor: (rowType === 'header') ? headerBackgroundColor : undefined
+                backgroundColor: (rowType === 'header') ? headerBackgroundColor : undefined,
+                padding: 0
```

Comment:
I think we can omit passing this parameter, since cell creation does not depend on the logic of this method.
Something like this:
```js
function generateRowCells({ dataProvider, rowIndex, wordWrapEnabled, colCount, backgroundColor }) {
    const result = [];
    for(let cellIndex = 0; cellIndex < colCount; cellIndex++) {
        const cellData = dataProvider.getCellData(rowIndex, cellIndex, true);
        const cellInfo = {
            gridCell: cellData.cellSourceData,
            pdfCell: {
                text: cellData.value,
                wordWrapEnabled,
                backgroundColor,
                padding: 0,
                _rect: {}
            }
        };
```

It's more clear, and other parameters like verticalAlign/horizontalAlign will be placed there too.